tests/basic/storage_pool_test.go

```diff
@@ -6315,3 +6315,88 @@ var _ = Describe("{PoolResizeInvalidPoolID}", func() {
 		AfterEachTest(contexts, testrailID, runID)
 	})
 })
+
+var _ = Describe("{PoolResizeReduceErrorcheck}", func() {
+	var testrailID = 79487
+	// Testrail Description : Resize with invalid pool size less than existing
+	// Testrail Corresponds : https://portworx.testrail.net/index.php?/cases/view/84470
+	var runID int
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("PoolResizeReduceErrorcheck",
+			"Resize with invalid reduced pool size",
+			nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+
+	var contexts []*scheduler.Context
+	stepLog := "Resize with reduce size than existing"
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+
+		startTime := time.Now()
+
+		// Get alerts from ten hours before current time till current time to get the sufficient alerts
+		startMinusTenHours := startTime.Add(time.Duration(-600) * time.Minute)
+		endTime := time.Now()
+		alertsBefore, err := Inst().V.GetAlertsUsingResourceTypeByTime(api.ResourceType_RESOURCE_TYPE_POOL, startMinusTenHours, endTime)
+		log.FailOnError(err, "Failed to fetch alerts between startTime [%v] and endTime [%v]", startMinusTenHours, endTime)
+
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("invalidpoolid-%d", i))...)
+		}
+		ValidateApplications(contexts)
+		defer appsValidateAndDestroy(contexts)
+
+		// Get the Pool UUID on which IO is running
+		poolUUID, err := GetPoolIDWithIOs()
+		log.FailOnError(err, "Failed to get pool using UUID [%v]", poolUUID)
+		log.InfoD("Pool UUID on which IO is running [%s]", poolUUID)
+
+		nodeDetail, err := GetNodeWithGivenPoolID(poolUUID)
+		log.FailOnError(err, "Failed to get Node Details from PoolUUID [%v]", poolUUID)
+
+		// Resize Pool with lower pool size than existing
+		// Do Pool Expand on the Node
+		stepLog = fmt.Sprintf("Expanding pool on node [%s] and pool UUID: [%s] using auto", nodeDetail.Name, poolUUID)
```

Comment:
update print statement as "Resizing pool on node [%s] and pool UUID: [%s] using auto", nodeDetail.Name, poolUUID"

Reply:
Done